module-services/service-time/AlarmServiceAPI.cpp

```diff
@@ -74,6 +73,26 @@ namespace alarms
         {
             return sendRequest<RegisterActiveAlarmsIndicatorHandlerRequestMessage>(serv);
         }
+
+        bool requestGetPreWakeUpStatus(sys::Service *serv, bool &response)
+        {
+            auto msg = std::make_shared<GetPreWakeUpRequestMessage>();
+            auto ret = serv->bus.sendUnicastSync(msg, service::name::service_time, 1000);
+
+            if (ret.first == sys::ReturnCodes::Success) {
+                auto getPreWakeUpResponse = std::dynamic_pointer_cast<GetPreWakeUpResponseMessage>(ret.second);
+                if ((getPreWakeUpResponse != nullptr) && (getPreWakeUpResponse->retCode == sys::ReturnCodes::Success)) {
+                    LOG_DEBUG("Is Pre-wake up active: %s", getPreWakeUpResponse->active ? "yes" : "no");
```

Comment:
this entry generates a lot of spam in the logs regardless of whether the alarm is set or not